README.md

```diff
@@ -18,7 +18,10 @@ Once you get your license, run our image with:
     docker run -e ADV_HOST=127.0.0.1 -e LICENSE_URL="[CHECK_YOUR_EMAIL_FOR_PERSONAL_ID]" \
                -p 3030:3030 -p 9092:9092 -p 2181:2181 -p 8081:8081 --name=lenses landoop/kafka-lenses-dev
 
-Once the services are loaded (it usually takes 30-45 seconds), visit http://localhost:3030 and login with `admin` / `admin`. If you are on macOS, depending on how you installed docker, you may have to set ADV_HOST to 192.168.99.100 and visit http://192.168.99.100:3030 instead.
+Once the services are loaded (it usually takes 30-45 seconds), visit http://localhost:3030 and login with `admin` / `admin` for full access
+or `logviewer` / `logviewer` for read-only access.
```

Comment:
Read-only access → limited access to certain topics. :)

Reply:
Revised, now displays "or `logviewer` / `logviewer` for  limited access to certain topics."